packages/mui-material/src/Button/Button.js

```diff
@@ -343,8 +391,120 @@ const ButtonEndIcon = styled('span', {
         marginRight: -2,
       },
     },
+    {
+      props: { loadingPosition: 'end', loading: true },
+      style: {
+        transition: theme.transitions.create(['opacity'], {
+          duration: theme.transitions.duration.short,
+        }),
+        opacity: 0,
+      },
+    },
+    {
+      props: { loadingPosition: 'end', loading: true, fullWidth: true },
+      style: {
+        marginLeft: -8,
+        order: 2,
```

Comment:
What are we using `order` for?

Reply:
Thanks, it exists before to fix the loader position. However, it does not need since we have the loader wrapper.

It's removed.